zerver/lib/actions.py

```diff
@@ -3737,10 +3737,14 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
 
 
 def do_delete_message(user_profile: UserProfile, message: Message) -> None:
+    print(message)
     event = {
         'type': 'delete_message',
         'sender': user_profile.email,
-        'message_id': message.id}  # type: Dict[str, Any]
+        'message_id': message.id,
+        'is_stream_message': message.is_stream_message(),
```

Comment:
It would be better to call this `message_type` and have it be either `stream` or `private, to follow the convention of how we detail this sort of thing elsewhere.